index.html

```diff
@@ -0,0 +1,152 @@
+            <div class="teams__div">
+                <div class="card">
+                    <img class="card__image" src="./images/doc1.png">
+                    <label class="card__name text--bold">
+                        Julian Jameson
+                    </label>
+                    <label class="card__proffesion text--muted">
+                        Proffesion
+                    </label>
+                    <div class="card__socials">
+                        <img src="./images/fb.png">
+                        <img src="./images/ig.png">
+                        <img src="./images/tw.png">
+                    </div>
+                </div>
+                <div class="card">
+                    <img class="card__image" src="./images/doc2.png">
+                    <label class="card__name text--bold">
+                        Julian Jameson
+                    </label>
+                    <label class="card__proffesion text--muted">
+                        Proffesion
+                    </label>
+                    <div class="card__socials">
+                        <img src="./images/fb.png">
+                        <img src="./images/ig.png">
+                        <img src="./images/tw.png">
+                    </div>
+                </div>
+            </div>
+            <div class="teams__div">
+                <div class="card">
+                    <img class="card__image" src="./images/doc3.png">
+                    <label class="card__name text--bold">
+                        Julian Jameson
+                    </label>
+                    <label class="card__proffesion text--muted">
+                        Proffesion
+                    </label>
+                    <div class="card__socials">
+                        <img src="./images/fb.png">
+                        <img src="./images/ig.png">
+                        <img src="./images/tw.png">
+                    </div>
+                </div>
+                <div class="card">
+                    <img class="card__image" src="./images/doc4.png">
+                    <label class="card__name text--bold">
+                        Julian Jameson
+                    </label>
+                    <label class="card__proffesion text--muted">
+                        Proffesion
+                    </label>
+                    <div class="card__socials">
+                        <img src="./images/fb.png">
+                        <img src="./images/ig.png">
+                        <img src="./images/tw.png">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="footer">
+        <label class="welcome__text--blue">Newsletter</label>
+        <label class="welcome__text--big">JOIN US</label>
+        <p class="footer__text doctor__text--muted">Problems trying to resolve the conflict between 
+            the two major realms of Classical physics: Newtonian mechanics </p>
```

Comment:
You are using modifiers without specifying the elements' classes. This is considered bad practice according to BEM naming convention. Perhaps do this `<label class="welcome__text welcome__text--blue"...`

Reply:
I still have some unresolved issues with the class inheritance! and Mixing